pandas/_libs/tslibs/timestamps.pyx

```diff
@@ -994,7 +994,7 @@ cdef class _Timestamp(ABCTimestamp):
 
         Parameters
         ----------
-        sep : str, default 'T'
+        sep : str, default 'min'
```

Comment:
looks like the docstring default changed but not the default in the signature?

Reply:
Oops, sorry, my mistake. I’ll correct the signature.